presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -207,6 +208,7 @@ public OrcWriter(
         requireNonNull(columnNames, "columnNames is null");
         requireNonNull(inputOrcTypes, "inputOrcTypes is null");
         this.orcTypes = inputOrcTypes.orElseGet(() -> OrcType.createOrcRowType(0, columnNames, types));
+        this.nodeIdToColumn = OrcType.createNodeIdToColumnMap(this.orcTypes);
```

Comment:
Do static import for createNodeIdToColumnMap (Option + Enter -> Add static import)